plugins/out_pgsql/pgsql_connections.c

```diff
@@ -73,6 +75,22 @@ void *pgsql_create_connection(struct flb_pgsql_config *ctx)
     flb_plg_info(ctx->ins, "switching postgresql connection "
                  "to non-blocking mode");
 
+    if (ctx->schema != NULL) {
+        escaped_schema = PQescapeLiteral(conn->conn, ctx->schema, flb_sds_len(ctx->schema));
+        res = PQexecParams(conn->conn,
```

Comment:
This query is not working, why not use PQexec() which is more simpler? PQexecParams() here is lacking of the proper parameters and is going out in errors

Reply:
Use PQexec instead of PQexecParams. How do you think?

https://github.com/fluent/fluent-bit/pull/5450/commits/c7178953bea4cd5a597f004084db86390c97e05e

Reply:
@kuroponzu still having the same issue

```
[2022/06/27 08:32:33] [2022/06/27 08:32:33] [ info] [output:pgsql:pgsql.0] switching postgresql connection to non-blocking mode
[engine] caught signal (SIGSEGV)
#0  0x7f82af888501      in  __vsnprintf_internal() at /x86_64-linux-gnu/11/include/stddef.h:112
#1  0x7f82af8608d5      in  __GI___snprintf() at �:31
#2  0x55aaf10f6067      in  pgsql_create_connection() at plugins/out_pgsql/pgsql_connections.c:83
#3  0x55aaf10f62d0      in  pgsql_start_connections() at plugins/out_pgsql/pgsql_connections.c:117
#4  0x55aaf10f4c25      in  cb_pgsql_init() at plugins/out_pgsql/pgsql.c:180
#5  0x55aaf100fbb6      in  flb_output_init_all() at src/flb_output.c:1066
#6  0x55aaf102159a      in  flb_engine_start() at src/flb_engine.c:668
#7  0x55aaf0ffcdce      in  flb_lib_worker() at src/flb_lib.c:626
#8  0x7f82af894b42      in  start_thread() at nix/sysv/linux/rseq-internal.h:442
#9  0x7f82af9269ff      in  ???() at one3.S:81
#10 0xffffffffffffffff  in  ???() at ???:0
Aborted (core dumped)
```
You're not using the proper rmalloc()